File: Beecrowd/Resolvidos/BEE1047.py
from math import floor
lista = input().split()
a = int(lista[0])
b = int(lista[1])
c = int(lista[2])
d = int(lista[3])
min1 = a*60 + b
min2 = c*60 + d
if a == b == c == d:
    print("O JOGO DUROU 24 HORA(S) E 0 MINUTO(S)")
elif min1 < min2:
    dif = min2 - min1
    h = dif/60
    mints = (dif % 60)
    print('O JOGO DUROU {} HORA(S) E {} MINUTO(S)'.format(floor(h), mints))
elif a == c:
    dif = b - d
    h =

File: Beecrowd/Organizar/BEE1047.cpp
#include <iostream>
using namespace std;
int main()
{
	int a, b, c, d, mini, minf, mints;
	cin >> a >> b >> c >> d;
	mini = a * 60 + b;	 // minuto inicial
	minf = c * 60 + d;	 // minuto final
	mints = mini - minf; // qnts minutos
	if (mini == minf)
	{
		cout << "O JOGO DUROU 24 HORA(S) E 0 MINUTO(S)" << endl;
	}
	else
	{
		if (mini < minf)
		{
			mints = minf - mini; // qnts minutos
			cout << "O JOGO DUROU " << mints / 60 << " HORA(S) E " << mints % 60 << " MINUTO(S)" << endl;
		}
		else
		{
			mints = (minf - mini) + 24 * 60; // qnts minutos
			cout << "O JOGO DUROU " << mints / 60 << " HORA(S) E " << mints % 60 << " MINUTO(S)" << endl;
		}
	}
}